index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Playfair+Display" />
+  <link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Poppins" />
+</head>
+<body>
+  <header>
+    <div id="burger-container">
+      <div id="burger-bar-1" class="burger-element"></div>
+      <div id="burger-bar-2" class="burger-element"></div>
+      <div id="burger-bar-3" class="burger-element"></div>
+    </div>
+    <h1>Charlie Giorgania</h1>
+    <p>
+      Hello! I am a software developer! I can help you build a product, feature
+      or website. Taka a look of my works. If you like what you see and have a
+      project you need coded, don’t hesitate and contact me.
+    </p>
+  </header>
+    <div id="collaboration">
+      <a id="collab-link" href="#" rel="noopener noreferrer">
+        <div id="collab-button">Start Collaboration</div>
+      </a>
+      <img id="mosaic-img" src="/Media/image_geometry_2.svg" alt=" ">
```

Comment:
- [ ] always add` ./` before the path to let the browser know the exact location of the file.

Reply:
Change request satisfied.